### PR TITLE
Fix missing Hello Minikube example in Spanish documentation

### LINKED ISSUE #20758: Localize “Hello Minikube” into Spanish
**This is a Bug Report**

**Problem:**
https://kubernetes.io/es/docs/tutorials/hello-minikube/ is a missing page; however, https://kubernetes.io/docs/tutorials/hello-minikube/ is part of the [required content](https://kubernetes.io/docs/contribute/localization/#translating-content) for a localization.

**Proposed Solution:**
Localize “Hello Minikube” into Spanish

**Page to Update:**

https://kubernetes.io/es/docs/tutorials/hello-minikube/

**Additional Information:**
/language es
/kind bug

### PR DESCRIPTION
**Pull request by qckzr**
I added the missing hello minikube example translated to Spanish (I'm native speaker).

This PR fix #20758

### CODE REVIEW

File: content/es/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+    El resultado es similar a:
+
+    ```
+    metrics-server was successfully disabled
+    ```
+
+## Limpieza
+
+Ahora se puede eliminar los recursos creados en el cluster:
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:248]:
```suggestion
Ahora se puede eliminar los recursos creados en el clúster:
```

File: content/es/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+```shell
+kubectl delete service hello-node
+kubectl delete deployment hello-node
+```
+
+Opcional, detener la máquina virutal Minikube:
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:255]:
```suggestion
Opcional, detener la máquina virtual de Minikube:
```

File: content/es/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+```shell
+minikube stop
+```
+
+Opcional, eliminar la máquina virtual Minikube:
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:261]:
```suggestion
Opcional, eliminar la máquina virtual de Minikube:
```

File: content/es/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+```shell
+minikube delete
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Leer más sobre [Despliegues](/docs/concepts/workloads/controllers/deployment/).
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:271]:
```suggestion
* Leer más sobre [Deployments](/docs/concepts/workloads/controllers/deployment/).
```

File: content/es/docs/tutorials/hello-minikube.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+* Leer más sobre [Desplegando aplicaciones](/docs/tasks/run-application/run-stateless-application-deployment/).
+* Leer más sobre [Servicios](/docs/concepts/services-networking/service/).
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:273]:
```suggestion
* Leer más sobre [Services](/docs/concepts/services-networking/service/).
```

**Comment by qckzr** [comment]
Gracias por las sugerencias @electrocucaracha !

File: content/es/docs/tutorials/hello-minikube.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+
+    El resultado es similar a:
+
+    ```
+    NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    hello-node   LoadBalancer   10.108.144.78   <pending>     8080:30369/TCP   21s
+    kubernetes   ClusterIP      10.96.0.1       <none>        443/TCP          23m
+    ```
+
+    En los proveedores Cloud que soporten balanceadores de carga, una dirreción IP externa será provisionada para acceder al servicio, en Minikube, el tipo `LoadBalancer` permite que el servicio sea accesible a través del comando `minikube service`.
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:147]:
```suggestion
    Para los proveedores Cloud que soportan balanceadores de carga, una dirección IP externa será provisionada para acceder al servicio, en Minikube, el tipo `LoadBalancer` permite que el servicio sea accesible a través del comando `minikube service`.
```

File: content/es/docs/tutorials/hello-minikube.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+
+3. Ejecutar el siguiente comando:
+
+    ```shell
+    minikube service hello-node
+    ```
+
+4. Solo en el ambiente de Katacoda: Hacer clic sobre el símbolo +, y luego en **Select port to view on Host 1**.
+
+5. Solo en el ambiente de Katacoda: Anotar el puerto de 5 dígitos visualizado opuesto al valor de `8080` en el resultado de servicios. Este número de puerto es generado aleatoriamente y puede ser diferente al indicado en el ejemplo. Escribir el número de puerto en el cuadro de texto y hacer clic en Display Port. Usando el ejemplo anterior, usted escribiría `30369`.
```
Comment (electrocucaracha) [inline content/es/docs/tutorials/hello-minikube.md:157]:
```suggestion
5. Solo en el ambiente de Katacoda: Anotar el puerto de 5 dígitos ubicado al lado del valor de `8080` en el resultado de servicios. Este número de puerto es generado aleatoriamente y puede ser diferente al indicado en el ejemplo. Escribir el número de puerto en el cuadro de texto y hacer clic en Display Port. Usando el ejemplo anterior, usted escribiría `30369`.
```

**Pull request merged**